frontend/scss/pages/_home_page.scss

```diff
@@ -199,6 +199,48 @@
   margin-top: 50px;
 }
 
+.products-component-container {
+  margin: 0 auto;
```

Comment:
if you add `text-align: center;` to this selector it will center the products because this is the parent. Just pointing it out since you asked about it